mvlearn/datasets/gaussian_mixture.py

```diff
@@ -93,101 +95,113 @@ def __init__(self, mu, sigma, n, class_probs=None, random_state=None,
 
         Attributes
         ----------
-        latent : np.ndarray
-            Latent distribution data. latent[:,i] is randomly sampled from
-            a gaussian distribution with mean mu[i] and covariance sigma[i].
-        Xs : list of array-like
+        latent_ : np.ndarray, of shape (n_samples, n_dims)
+            Latent distribution data. latent[i] is randomly sampled from
+            a gaussian distribution with mean centers[i] and covariance
+            covariances[i].
+        y_ : np.ndarray, of shape (n_samples)
```

Comment:
This is probably intuitive what y is, but it's the only attribute without a description